lib/src/infinite_list.dart

```diff
@@ -76,6 +77,12 @@ class InfiniteList extends StatelessWidget {
   /// {@endtemplate}
   final bool reverse;
 
+  /// {@template shrinkWrap}
```

Comment:
Nit: If the `@template` is not being used elsewhere in a `@macro` we should remove the `@template`.